src/Illuminate/Database/Schema/Builder.php

```diff
@@ -298,12 +298,15 @@ public function hasColumns($table, array $columns)
      * @param  string  $table
      * @param  string  $column
      * @param  \Closure  $callback
+     * @param  \Closure|null  $otherwise
```

Comment:
You could narrow the `Closure` further down by adding argument types:

```suggestion
    * @param  \Closure(Blueprint): mixed  $callback
     * @param  (\Closure(Blueprint): mixed)|null  $otherwise
```